scripts/ecosystem_fix_all_check.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This is @konstin's setup for checking an entire checkout of ~2.1k packages for
+# panics, autofix errors and similar problems.
+#
+# We put this in a docker container because processing random scraped code from GitHub is
+# [kinda dangerous](https://moyix.blogspot.com/2022/09/someones-been-messing-with-my-subnormals.html)
+
+# https://stackoverflow.com/a/246128/3549270
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+time docker run --rm -it \
+  -w /app \
+  -v "${SCRIPT_DIR}/../target/checkouts:/app/checkouts" \
+  -v "${SCRIPT_DIR}/../target/ecosystem_fix_all_results:/app/ecosystem_fix_all_results" \
+  -v "${SCRIPT_DIR}/../target/release/ruff:/app/ruff" \
+  -v "${SCRIPT_DIR}/../ecosystem_all.py:/app/ecosystem_all.py" \
+  -v "${SCRIPT_DIR}/../github_search.jsonl:/app/github_search.jsonl" \
+  -v "${SCRIPT_DIR}/../.venv-3.11:/app/.venv" \
```

Comment:
I think it'd be useful to briefly document the expected workflow for running this, or at least the assumptions. At a glance, I see that:

1. It assumes you have a `.venv-3.11` installed in your root directory.
2. It assumes that you have `ripgrep` installed.
3. ...anything else? `checkouts`?

Reply:
The venv is created by the script, i replace ripgrep with grep. The checkouts need to come from the ecosystem ci, so it's a bit more involved to run